### PR TITLE
modify some codes in cloudhub

### PR DESCRIPTION
**Pull request by gy95**
Signed-off-by: gy95 <guoyao17@huawei.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
1、emphasize error is nil, so return nil directively even though `err` is equal to nil
2、`ExtractMessage`: check whether context is legal before parsing the message

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by Iceber — COMMENTED** [review]
LGTM overall
/lgtm

File: cloud/pkg/cloudhub/servers/udsserver/server.go [hunk 1]
```diff
@@ -51,16 +51,15 @@ func StartServer(address string) {
...
-			return nil, err
-		}
-	} else {
-		err := errors.New("failed with error: context is empty")
-		klog.Errorf("%v", err)
+	if context == "" {
+		return nil, errors.New("failed with error: context is empty")
+	}
+
+	err := json.Unmarshal([]byte(context), &msg)
```
[the remark this answers is not in the document]
Comment (gy95) [reply cloud/pkg/cloudhub/servers/udsserver/server.go:58]: ok

**Review by fisherxu — COMMENTED** [review]
Please squash the commits.

**Comment by gy95** [comment]
> Please squash the commits.

ok

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**